platform/qt/src/qmapboxgl.cpp

```diff
@@ -1705,14 +1705,14 @@ void QMapboxGL::connectionEstablished()
     \a copyrightsHtml is a string with a HTML snippet.
 */
 
-mbgl::MapOptions mapOptionsFromQMapboxGLSettings(const QMapboxGLSettings &settings) {
+mbgl::MapOptions& mapOptionsFromQMapboxGLSettings(const QMapboxGLSettings &settings) {
```

Comment:
why return reference (to a temporary?) ?

Reply:
Sorry - I think it is a leftover from my experiments with the options being movable-only, will remove.